docs/docusaurus.config.js

```diff
@@ -159,6 +157,11 @@ const config = {
         defaultMode: 'light',
         disableSwitch: true,
       },
+      algolia: {
+        appId: 'WR5FASX5ED',
+        apiKey: '299e4601d2fc5d0031bf9a0223c7f0c5',
```

Comment:
https://docusaurus.io/docs/search

Says that it is safe to commit this, which feels weird.

Reply:
It could be similar to how you can publish your google analytics id live and it isn't an issue? Hmm

Reply:
is there a way for us to embedded these keys via the env? Personally don't think it is a good idea to just have these available for someone to just copy

Reply:
I think this is ok, as I believe this is just a reference to the search index (this can probably be extracted fairly easily from the built assets, too)